msteams-platform/includes/get-started/azure-provisioning-instructions-tab.md

```diff
@@ -24,26 +24,28 @@ Select the Teams Toolkit :::image type="icon" source="~/assets/images/teams-tool
 
    > [!NOTE]
    > There are always some Azure resources used for hosting your app.
```

Comment:
If we are trying to say that apps/resources are always hosted in Azure then this sentence needs an update to convey it in better words.

Reply:
Note has been changed to " Your app will always be hosted using certain Azure resources.